airflow/providers/openlineage/utils/utils.py

```diff
@@ -395,3 +397,10 @@ def wrapper(*args, **kwargs):
             log.exception(e)
 
     return wrapper
+
+
+def is_source_enabled():
+    source_var = conf.get(
+        "openlinege", "disable_source_code", fallback=os.getenv("OPENLINEAGE_AIRFLOW_DISABLE_SOURCE_CODE")
+    )
+    return isinstance(source_var, str) and source_var.lower() not in ("true", "1", "t")
```

Comment:
```suggestion
    return conf.getboolean(
        "openlinege",
        "disable_source_code",
        fallback=os.getenv("OPENLINEAGE_AIRFLOW_DISABLE_SOURCE_CODE", "").lower() not in ("true", "1", "t")
    )
```

Reply:
Can't do this this way unfortunately, since `conf.getboolean` needs default value, and default value prevents fallback from working, since it's taken before fallback.